pyccel/codegen/pipeline.py

```diff
@@ -239,6 +243,9 @@ def get_module_dependencies(parser, mods=(), folders=()):
                                 includes=includes,
                                 libdirs=libdirs)
 
+        if not codegen.is_program and language == "c":
+            if sys.platform == "darwin":
+                flags = flags.replace("-lomp", "")
```

Comment:
yes, it give a waring when compiling: unused argument.

Reply:
As long as it is just a warning I would leave it. It can be fixed with issue #489.
As you can see in the output on travis we have the same problem with the math library:
```
tests/epyccel/test_base.py: 27 warnings
tests/epyccel/test_bitwise.py: 37 warnings
tests/epyccel/test_epyccel_IfTernaryOperator.py: 6 warnings
tests/epyccel/test_epyccel_default_args.py: 4 warnings
tests/epyccel/test_epyccel_division.py: 13 warnings
tests/epyccel/test_epyccel_functions.py: 17 warnings
tests/epyccel/test_epyccel_mod.py: 5 warnings
tests/epyccel/test_epyccel_optional_args.py: 4 warnings
tests/epyccel/test_epyccel_pow.py: 9 warnings
tests/epyccel/test_kind.py: 6 warnings
tests/epyccel/test_loops.py: 9 warnings
tests/epyccel/recognised_functions/test_math_funcs.py: 78 warnings
tests/epyccel/recognised_functions/test_numpy_funcs.py: 68 warnings
  /usr/local/lib/python3.8/site-packages/pyccel/codegen/utilities.py:145: UserWarning: clang: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]
```
I think we should do the same thing with this linker flag as we did with the math flag. In this case that means that we should leave it to help with debugging #489 rather than adding a somewhat hacky solution as a temporary fix that just hides the problem